composer.json

```diff
@@ -34,6 +34,7 @@
         "justbetter/statamic-eloquent-driver-globalset-migration-generator": "^0.1.0",
         "rapidez/blade-directives": "^0.6",
         "justbetter/statamic-glide-directive": "^2.1",
+        "spatie/once": "*",
```

Comment:
This is added in Laravel 11: https://laravel.com/docs/11.x/releases#the-once-function, active support for Laravel 10 ended, see: https://endoflife.date/laravel we could move on but maybe we need to require `illuminate/support:^11.0`

Reply:
We don't necessarily need to require L11+ for this functionality.
If we require it like this it'll install `spatie/once` for <11 and `illuminate/support` will overwrite it on >=11
https://packagist.org/packages/illuminate/support#:~:text=collect%3A%20%3C5.5.33-,replaces,-spatie/once%3A%20*